tests/retrolab/README.md

```diff
@@ -0,0 +1,31 @@
+# Retrolab accessibility tests
+
+## :package: Requirements
+
+To run the tests in this directory you need the following pre-requisites:
+
+- python > 3.7
+- nox
+- pyyaml
+
+you can install both nox and pyyaml from the command line with the following command:
+
+```bash
+python3 -m  pip install nox pyyaml
+```
+
+## :zap: Running the tests
+
+1. Make sure you are in the correct directory:
+
+    ```bash
+    cd tests/retrolab
+    ```
+
+2. Run the test with snox from the command line:
```

Comment:
```suggestion
2. Run the test with nox from the command line:
```

now you've got me wondering what a snox would be if a snox were a thing... sounds like something from Dr. Seuss or Roald Dahl might come up with...